applications/API/serializers.py

```
@@ -0,0 +1,576 @@
+        if isinstance(value, str):
+            value = cls._normalize_category_input(value)
+
+        # First try as category key
+        try:
+            category_name = cls.cacher.get_category_name_by_key(value)
+            validated = {
+                "category_key": value,
+                "category_name": category_name,
+            }
+            logger.debug(f"Serializer Resolved {value} to {validated}")
+            return validated
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)